src/migrations/user-store/5.0.0-alpha.2.ts

```diff
@@ -0,0 +1,18 @@
+// Switch representation of hiddenTableColumns in store
+import { migration } from "../migration-wrapper";
+
+export default migration({
+  version: "5.0.0-alpha.2",
+  run(store) {
+    const preferences = store.get("preferences");
+    const oldHiddenTableColumns: Record<string, string[]> = preferences?.hiddenTableColumns;
+
+    if (!oldHiddenTableColumns) {
+      return;
+    }
+
+    preferences.hiddenTableColumns = Object.entries(oldHiddenTableColumns);
+
+    store.get("preferences", preferences);
```

Comment:
`store.set`?  (deja vu)

Reply:
This needs to be fixed.

Reply:
Agreed, I am sorry that this slipped in. It almost certainly was the result of rebasing against the wrong branch target.